packages/calcite-components/src/utils/floating-ui.ts

```diff
@@ -153,13 +153,20 @@ export const positionFloatingUI =
 
     const { open } = component;
 
+    if (!open) {
+      Object.assign(floatingEl.style, hiddenFloatingElStyle);
```

Comment:
If for some reason the component is no longer open during a debounced reposition, we can assign the hidden styles and stop.